packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -5,7 +5,7 @@ import { ViewProps } from 'react-native';
 import { IRenderData } from '@uifabricshared/foundation-composable';
 import { ITextProps } from '@fluentui-react-native/text';
 import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
-import { IViewWin32Props } from '@office-iss/react-native-win32';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
```

Comment:
What does adding type do here? But also, could we just use adapters here?

Reply:
It specifies that you are only importing type information, no actual JS.  In this case it shouldn't make any difference since IViewWin32Props is only a type.  But it makes it more explicit that its just a type information import.